TheMovieDbTests/TheMovieDbTests.swift

```diff
@@ -9,16 +9,27 @@ import XCTest
 @testable import TheMovieDb
 
 class TheMovieDbTests: XCTestCase {
+    var sut: MovieClient!
 
     override func setUpWithError() throws {
         // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        try super.setUpWithError()
```

Comment:
TheMovieDbTests do not subclass custom XCTestCase, super call may be unnecessary